### PR TITLE
Simplify decoding of Synthetix/Kwenta markets and assets. Remove extra characters/spaces from assets.

### PR DESCRIPTION
**Pull request by RplusT**
Brief comments on the purpose of your changes:
This PR removes the extra characters/spaces from Synthetix/Kwenta in the `perpetual.trades` table. The extra characters will be a problem when it comes to joining on market and asset columns, and also when aggregating assets and markets across the different perpetuals trading protocols. The decoding for these markets and assets have also been simplified for easier reviews later on.

*For Dune Engine V2*
I've checked that:
General checks:
* [X] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [X] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [X] the filename is unique and ends with .sql
* [X] each sql file is a select statement and has only one view, table or function defined  
* [X] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributors)
* [ ] if adding a new materialized table, I edited the optimize table macro

Join logic:
* [X] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

Incremental logic:
* [X] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
* [X] where block_time >= date_trunc("day", now() - interval '1 week')
* [X] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Comment by soispoke** [comment]
lgtm, thanks @RplusT !

**Review by soispoke — APPROVED** [review]

**Comment by RplusT** [comment]
Thanks @soispoke! Checked the table just now on [dune.com](https://dune.com/queries/1344052?d=1) and it seems the only Synthetix transactions that were updated with the corrected spacing are the ones from September 26 onwards. Would you have an idea why this is the case? Something related to the incremental updating perhaps?

**Comment by soispoke** [comment]
@RplusT Hmm weird, I did a full-refresh on this table, let me try again and let you know how it goes

**Comment by RplusT** [comment]
Hi @soispoke! Just wanted to ask if there's an update on this? :)

**Comment by soispoke** [comment]
@RplusT Hi, sorry I did run a full refresh again, did that not solve anything on your end ?

**Comment by RplusT** [comment]
@soispoke still the same problem unfortunately. The previously decoded markets and underlying assets still have those "hidden spaces" up until the September 26 data.

**Comment by soispoke** [comment]
@RplusT I ran it again, if it doesn't work I think it must come from the spell itself

**Comment by RplusT** [comment]
@soispoke Got it. I did run the Synthetix-specific spell manually on [dune.com](https://dune.com/queries/1214339) (without the incremental conditions of course) and the extra spaces are gone. Perhaps it's some other part of the whole set of perpetuals spells but yeah, will try to dig.

**Comment by RplusT** [comment]
@soispoke just to be clear, the full refresh was ran on the `synthetix_optimism_trades` model itself, right?

**Comment by soispoke** [comment]
@RplusT right!

**Comment by RplusT** [comment]
@soispoke got it. So I think I get the problem now. Could you also run a full refresh on the `perpetual.trades` table itself? :)

**Comment by RplusT** [comment]
The `synthetix_optimism_trades` model was refreshed but the changes didn't reflect downstream to the rest of the consolidated `perpetual.trades` model (and I guess the associated models).

**Comment by soispoke** [comment]
@RplusT yes, got it :) Should be good now ?

**Comment by RplusT** [comment]
@soispoke awesome, it looks good now! thank you so much for being responsive and patient. 😅

**Comment by soispoke** [comment]
@RplusT ofc! Happy to help out thanks a lot for your contributions !

**Comment by jeff-dude** [comment]
thanks for fixing this @RplusT after we struggled in initial build -- this could be useful for others in the future! will be sure to remember this model as reference if the situation arises again

**Comment by RplusT** [comment]
@jeff-dude haha thanks for spotting the problem in the first place! I couldn't live with those extra spaces. 😅
